server/grpc/gogoreflection/fix_registration.go

```diff
@@ -1,90 +1,15 @@
 package gogoreflection
```

Comment:
@testinginprod What do you think to even completely remove this package? We have the runtime reflection service, so maybe gRPC native reflection is not needed anymore?

Reply:
Talked offline, let's support grpc native reflection